chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -332,6 +334,19 @@ void OnRewriteSuggestionCompleted(
 }
 #endif  // BUILDFLAG(ENABLE_AI_CHAT)
 
+bool CanSplitViewEnabled(base::WeakPtr<content::WebContents> web_contents) {
+  Browser* browser = chrome::FindBrowserWithTab(web_contents.get());
+
+  return base::FeatureList::IsEnabled(tabs::features::kBraveSplitView) &&
+         brave::CanOpenNewSplitViewForTab(browser);
```

Comment:
I think we can early return before find browser with contentents.

```suggestion
  if (!base::FeatureList::IsEnabled(tabs::features::kBraveSplitView)) {
    return false;
  }
  Browser* browser = chrome::FindBrowserWithTab(web_contents.get());
  return brave::CanOpenNewSplitViewForTab(browser);
```